configs/raspberrypi_config

```diff
@@ -9,13 +9,12 @@ RASPBERRYPI_BINARIES_URL="${MENDER_STORAGE_URL}/mender-convert/uboot/raspberrypi
 RASPBERRYPI_OS_REQUIRED_ID="raspbian"   # historical distro name of the Raspberry Pi OS Debian derivative
 RASPBERRYPI_OS_MINIMAL_VERSION=12       # this is "bookworm" or later, as it changed the boot partition mount directory
 
+RPI_NEEDS_32BIT="y"
```

Comment:
Nit pick: for consistency with the above variables, use `RASPBERRYPI_OS_NEEDS_32BIT`